py_outbrain/services/base.py

```diff
@@ -24,6 +24,19 @@ def execute(self, method, uri, query_params=None, **payload):
         return self.client.execute(method, uri,
                                    query_params=query_params, **payload)
 
+    def get_child_elements(self, element_id, child_endpoint, params):
+        if 'offset' not in params:
+            params['offset'] = 0
+        url = '{}/{}'.format(element_id, child_endpoint)
+        do_continue = True
+        while do_continue:
```

Comment:
do_continue is never altered, right ?
```suggestion
        while True:
```